src/convertToObject.js

```diff
@@ -1,18 +1,26 @@
 'use strict';
 
 /**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
+    .split(';')
+    .map((property) => property
+      .trim()
+      .split(':')
+      .map((word) => word.trim()))
+    .filter((isNotSpace) => isNotSpace.length > 1);
+  const objectOfStyles = {};
+
+  for (const property of styles) {
```

Comment:
<img width="635" alt="image" src="https://github.com/mate-academy/js_style-to-object/assets/86789161/d49a52d0-6eda-4e70-b162-8ad1be2ccbd0">